### ISSUE TITLE
[A11y] Desktop/Mobile Work Show Page-CPD Child: All 5 image thumbnails

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1952]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Work Show

### Admin user?

No

### Issue Description

Work Show Page-CPD Child: https://staging.oregondigital.org/concern/documents/df727057n?locale=en

All 5 thumbnail images are showing the following failed instance:
image-alt: Ensures <img> elements have alternate text or a role of none or presentation

Path: 
.row:nth-child(9) > .col-sm-12 > .hyc-bl-results.hyc-bl-search.row > .row > .masonry.col-xs-6.col-md-3 > .thumbnail > a > img

Snippet:
`<img src="/assets/work-ff055336041c3f7d310ad69109eda4a887b16ec501f35afc0a547c4adb97ee72.png">`

### Suggested Fix

Fix ONE of the following:

- Element does not have an alt attribute
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

### Can anybody QA or does this need an expert?

A11y familiarity recommended

**Comment by CGillen** [comment]
Images contain alt text as far as I can tell. Fixed by implementing wireframes

**Comment by azle** [comment]
When I was validating the page the following error came up for link name

<img width="803" alt="Screen Shot 2022-01-19 at 10 49 03 AM" src="https://user-images.githubusercontent.com/8353860/150194893-201f61a3-b7e8-4ebc-bfbe-02c6dfe96352.png">

**Comment by CGillen** [comment]
Above issue is related to an UV issue we currently can not fix. Please disregard until we can edit the UV iframe (if we decide to)
